plugins/tech-insights-backend/src/service/persistence/TechInsightsDatabase.ts

```diff
@@ -168,6 +185,58 @@ export class TechInsightsDatabase implements TechInsightsStore {
     return existingSchemas.find(it => it.version === sorted[0])!!;
   }
 
+  private async deleteExpiredFactsByDate(
+    tx: Transaction,
+    factRows: { id: string; entity: string }[],
+    timestamp: DateTime,
+  ) {
+    await tx<RawDbFactRow>('facts')
+      .whereIn(
+        ['id', 'entity'],
+        factRows.map(it => [it.id, it.entity]),
+      )
+      .and.where('timestamp', '<', timestamp.toISO())
```

Comment:
We got bitten by a few comparisons differences in postgres and sqlite when building catalog. Just some thing to double check that it works as expected in both :) Perhaps the test suite take care of that nowadays but just wanted to share that as this plugin is quite involved with the database

Reply:
We luckily have test cases for these so it should be fine.